doc/modules/cassandra/pages/managing/operating/auto_repair.adoc

```diff
@@ -0,0 +1,410 @@
+
+Before enabling auto repair, please consult the xref:managing/operating/repair.adoc[Repair] guide to establish a base
+understanding of repairs.
+
+=== Full Repair
+
+Full Repairs operate over all data in the token range being repaired.  It is therefore important to run full repair
+with a longer schedule and with smaller assignments.
+
+=== Incremental Repair
+
+When enabled from the inception of a cluster, incremental repairs operate over unrepaired data and should finish
+quickly when run more frequently.
+
+Once incremental repair has been run, SSTables will be separated between data that have been incrementally repaired
+and data that have not.  Therefore, it is important to continually run incremental repair once it has been enabled so
+newly written data can be compacted together with previously repaired data, allowing overwritten and expired data to
+be eventually purged.
+
+Running incremental repair more frequently keeps the unrepaired set smaller and thus causes repairs to operate over
+a smaller set of data, so a shorter `min_repair_interval` such as `1h` is recommended for new clusters.
+
+NOTE: When turning on incremental repair for the first time on an existing cluser with a large amount of data it is
+advisable to use a larger interval such as `24h` or longer to reduce the impact of anticompaction caused by incremental
+repair.  Once the entire token range has been repaired, one may consider reducing this to a shorter interval.  It is
+strongly recommended to not enable incremental repair for an existing cluster unless full repair has previously
+actively run.
+
+=== Previewing Repaired Data
+
+The `preview_repaired` repair type executes repairs over the repaired data set to detect possible data inconsistencies.
+
+Inconsistencies in the repaired data set should not happen in practice and could indicate a possible bug in incremental
+repair.
+
+Running preview repairs is useful when considering using the
+xref:cassandra:managing/operating/compaction/tombstones.adoc#deletion[only_purge_repaired_tombstones] table compaction
+option to prevent data from possibly being resurrected when inconsistent replicas are missing tombstones from deletes.
+
+When enabled, the `BytesPreviewedDesynchronized` and `TokenRangesPreviewedDesynchronized`
+xref:cassandra:managing/operating/metrics.adoc#table-metrics[table metrics] can be used to detect inconsistencies in the
+repaired data set.
+
+== Configuring auto repair in cassandra.yaml
+
+Configuration for auto repair is managed in the `cassandra.yaml` file by the `auto_repair` property.
+
+A rich set of configuration exists for configuring auto repair with sensible defaults.  However, the expectation
+is that some tuning might be needed particulary when it comes to tuning how often repair should run
+(`min_repair_interval`) and how repair assignments as created (`token_range_splitter`).
+
+The following is a practical example of an auto_repair configuration that one might use.
+
+[source, yaml]
```

Comment:
Figured showing a minimal example would be useful to give some context to those only looking at the doc side, as the cassandra.yaml doc page doesn't inline commented out subproperties, only the top level.  The audit log documentation page does something similar (https://cassandra.apache.org/doc/5.0/cassandra/managing/operating/auditlogging.html#configuring-audit-logging-in-cassandra-yaml) since it also has nested properties.

Reply:
![image](https://github.com/user-attachments/assets/a0f71642-1d5c-41c9-96a0-0f998b78311d)